src/properties/module/index.test.js

```diff
@@ -152,6 +166,37 @@ const invalidModuleConfigs = [
     },
     schema: schemaFn({ rules: { 'loader-enforce-include-or-exclude': true } }),
   },
+  {
+    // should enforce `loader` property, if object is found in `loaders` array
+    input: {
+      loaders: [{
+        test: /foo/,
+        loaders: [
+          { query: { foo: 'bar' } },
+        ],
+      }],
+    },
+    error: {
+      message: '"loaders" at position 0 does not match any of the allowed types',
```

Comment:
Is there a relatively simple way to make this message indicate that the allowed types are objects and strings?